Formula/vivid.rb

```diff
@@ -0,0 +1,17 @@
+class Vivid < Formula
+  desc "Generator for LS_COLORS with support for multiple color themes"
+  homepage "https://github.com/sharkdp/vivid"
+  url "https://github.com/sharkdp/vivid/archive/v0.6.0.tar.gz"
+  sha256 "c8640f524aef1cd4dc15286bdc6189894ad067ea79bf8c40b9ca8d9d752d161f"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
```

Comment:
Probably want to run `cargo build --release` rather than cargo install

Then also add `bin.install "target/release/vivid"`

Reply:
We have pretty standardized around `cargo install`, what is the advantage of using `cargo build --release`?

Reply:
TBH, I'm not sure. I haven't seen enough Rust formulae to say which method is better, but I thought it made more sense to build from source since it wouldn't depend on crates.io

Reply:
from what I understand, `cargo install --locked` is essentially the same thing as `cargo build --release`.

Reply:
Huh. Did not know that. Where does it install the binary?

Reply:
The `std_cargo_args` also contain a `--root #{prefix}`. So after install everything is automatically where Homebrew likes it.

Reply:
yeah, exactly.